student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,5 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(my_username(),malloc_username_from_conf_file(),"ajsanthosh14");
```

Comment:
@ajsanthosh14  what will happen to the memory that is malloc for username if its not freed?

Reply:
@Maitreyee2095 I totally forgot to review the malloc_username_from_conf_file() call to verify it's being freed or not. I will take care of this from now on.
--> Usually, If we don't free this memory, It will get freed at the exit/end of the process/program.
--> There's also another possibility, If this process is like a daemon, which is running in the background may end up being responsible for memory leakage. (But luckily it's not!)